lib/sprockets/commoner/processor.rb

```diff
@@ -70,12 +67,13 @@ def call(input)
 
         babel_config = babelrc_data(filename)
 
-        result = input[:cache].fetch([filename, @cache_key, input[:data], babel_config]) do
-          transform(input[:data], options(input), paths: @env.paths)
-        end
+        result = transform(input[:data], options(input), paths: @env.paths)
```

Comment:
It's still caches on a file level just not the transform specifically. It can lead to bugs and is not the biggest turn spend I believe